ext/fs/ops.rs

```diff
@@ -382,6 +382,32 @@ where
   Ok(())
 }
 
+#[op2(stack_trace)]
+pub fn op_fs_fchown_sync(
+  state: &mut OpState,
+  #[smi] rid: ResourceId,
+  uid: Option<u32>,
+  gid: Option<u32>,
+) -> Result<(), FsOpsError> {
+  let file =
+    FileResource::get_file(state, rid).map_err(FsOpsErrorKind::Resource)?;
```

Comment:
Do we need to add permission checks here like in `op_fs_chown_async`?

Reply:
I'm not entirely sure, but other ops which take resource id as parameter generally don't seem checking permissions (e.g. [op_fs_ftruncate_sync](https://github.com/denoland/deno/blob/7abce12183f24c6d8c5a939392545c11fbc28f6a/ext/fs/ops.rs#L1625), [op_fs_futime_sync](https://github.com/denoland/deno/blob/7abce12183f24c6d8c5a939392545c11fbc28f6a/ext/fs/ops.rs#L1649), etc)

I think this is because the permission is checked when the file is opened (i.e. when fd obtained)

(Correct me if I'm wrong. cc @littledivy @bartlomieju )

Reply:
Yes, creating the file resource already checks for --allow-read